exercises/G-conditionals-2/exercise-1.js

```diff
@@ -6,9 +6,12 @@
   - if number is more or equal to zero, return the word "positive"
 */
 
-function negativeOrPositive(number) {}
+function negativeOrPositive(number) {
```

Comment:
You could get away with just an `if...else...` here instead of `if... else if....`. A number can only be either positive or negative.